docs/Guides/Porting.md

```diff
@@ -63,6 +63,8 @@ In the example above, assume the only installed version of `m4` is `${HOME}/zot/
 - `. ./env`
 to set up the environment for `m4`. If the `.env` file does not exist, the build will fail. 
 
+The `.env` files for each tool can be retrieved from running the `portcrtenv.sh` script that is provided either from the respective tool's repository, or the `utils` repository in the `env/` directory.
```

Comment:
you might want to mention that portcrtenv.sh is driven from build.sh as part of the 'install process' - but that it can be run stand-alone if required.

Reply:
Sounds good. I added some additional text.